pkg/kubernetes/helm.go

```diff
@@ -236,10 +227,15 @@ func (h *HelmExecute) RunHelmInstall() error {
 		helmParams = append(helmParams, "--debug")
 	}
 
-	h.utils.Stdout(h.stdout)
-	log.Entry().Info("Calling helm install ...")
-	log.Entry().Debugf("Helm parameters: %v", helmParams)
-	if err := h.utils.RunExecutable("helm", helmParams...); err != nil {
+	if h.verbose {
+		helmParamsDryRun := helmParams
+		helmParamsDryRun = append(helmParamsDryRun, "--dry-run")
+		if err := h.runHelmCommand(helmParamsDryRun); err != nil {
+			log.Entry().WithError(err).Fatal("Helm install --dry-run call failed")
```

Comment:
maybe not fail here, since `verbose` should help to identify issues and provide further insights.

I would suggest to use `.Error(` instead

Reply:
Done